### PR TITLE
Consolidate UICatalog tests to cut down on time outs

### PR DESCRIPTION
**Pull request by imurchie**
All the tests remain, but session creation is minimized.

### CODE REVIEW

File: test/e2e/uicatalog/find-specs.js [hunk 1]
```diff
@@ -0,0 +1,378 @@
+import env from '../helpers/env';
+import setup from "../setup-base";
+import desired from './desired';
+import B from 'bluebird';
+import _ from 'lodash';
+import { clickButton, throwMatchableError, filterDisplayed,
+         filterVisibleUiaSelector } from '../helpers/recipes';
+
+console.log(env.IOS8, env.IOS9);
```
Comment (jlipps) [inline test/e2e/uicatalog/find-specs.js]: do we want to leave this in with no context?

**Comment by jlipps** [comment]
again will trust you on the details here. +1 for shorter tests!

File: test/e2e/uicatalog/find-specs.js (hunk 1 above)
Comment (imurchie) [inline test/e2e/uicatalog/find-specs.js]: No. I need to clean this up. I just wanted to see how it handles in Travis (locally the whole UICatalog suite runs in 13 minutes now).

**Comment by imurchie** [comment]
UICatalog tests from 40 to 16-18 minutes!

**Pull request merged**